app_test.go

```diff
@@ -6,9 +6,10 @@ import (
 )
 
 var (
-	cfgdir = "./cfg"
-	ver    = "17.06"
-	path   string
+	cfgdir     = "./cfg"
+	ver        = "17.06"
+	path       string
+	configPath string
```

Comment:
I'm wondering if there is any reason for `path` and `configPath` to be global rather than local variables? Would be better if they are local to each test, because otherwise one test can influence another